Content.Shared/Humanoid/SharedHumanoidAppearanceSystem.cs

```diff
@@ -250,6 +250,26 @@ public void SetSex(EntityUid uid, Sex sex, bool sync = true, HumanoidAppearanceC
         }
     }
 
+    // Parkstation-HeightSlider Start
+    /// <summary>
+    ///     Set a humanoid mob's sex. This will not change their gender.
+    /// </summary>
+    /// <param name="uid">The humanoid mob's UID.</param>
+    /// <param name="height">The height to set the mob to.</param>
+    /// <param name="sync">Whether to immediately synchronize this to the humanoid mob, or not.</param>
+    /// <param name="humanoid">Humanoid component of the entity</param>
+    public void SetHeight(EntityUid uid, float height, bool sync = true, HumanoidAppearanceComponent? humanoid = null)
+    {
+        if (!Resolve(uid, ref humanoid) || Math.Abs(humanoid.Height - height) < 0.01f)
```

Comment:
Is there a reason we're checking for very small changes? Is there a problem with adjusting it when not necessary?

Reply:
It's supposed to check if it's the same but with how floats are I had to make it check for small changes

Reply:
I get that, but why do we care? Is there a problem with re-setting it? 
Also, double check if there's an aprox. equal to function. Those are pretty common when working with floats.

Reply:
> Is there a problem with re-setting it?

Not really, it's just nice not to send unnecessary data